### PR TITLE
OFFPAY-543: analytics code with customised virtual page

### PR DESCRIPTION
**Pull request by codesurf42**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -17,6 +17,7 @@ include "common.conf"
 appName="off-payroll-frontend"
 application.router=prod.Routes
 ui.debug = false
+appUrlPath = "check-employment-status-for-tax"
```
Comment (peteanning) [inline conf/application.conf]: Is there no way to get this dynamically form Prod.routes

Comment (codesurf42) [inline conf/application.conf]: I guess prod.routes would rather need to get it dynamically as it is hardcoded there now

File: app/uk/gov/hmrc/offpayroll/views/govuk_wrapper.scala.html [hunk 2]
```diff
@@ -37,13 +39,21 @@
 
 @afterHeader = {}
 
+@analyticsCode(virtualUrl: String, analyticsToken: String) = {
+  ga('create', '@analyticsToken', {'cookieDomain': 'https://www-dev.tax.service.gov.uk/'});
```
Comment (codesurf42) [inline app/uk/gov/hmrc/offpayroll/views/govuk_wrapper.scala.html]: only for test

File: conf/application.conf [hunk 3]
```diff
@@ -91,7 +92,7 @@ auditing {
 }
 
 google-analytics {
-  token=N/A
+  token=UA-43414424-24
```
Comment (codesurf42) [inline conf/application.conf]: test only

**Pull request merged**